scripts/LSSTComCam/runHeadNode.py

```diff
@@ -19,22 +19,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-import sys
-
 import lsst.daf.butler as dafButler
 from lsst.rubintv.production.processingControl import HeadProcessController
-from lsst.rubintv.production.utils import LocationConfig
+from lsst.rubintv.production.utils import getAutomaticLocationConfig
 from lsst.summit.utils.utils import setupLogging
 
-instrument = "LSSTComCam"
-
 setupLogging()
-location = "summit" if len(sys.argv) < 2 else sys.argv[1]
-locationConfig = LocationConfig(location)
-print(f"Running {instrument} head node at {location}...")
-
+instrument = "LSSTComCam"
+locationConfig = getAutomaticLocationConfig()
+print(f"Running {instrument} head node at {locationConfig.location}...")
 
-butler = dafButler.Butler(
+butler = dafButler.Butler(  # type: ignore
```

Comment:
You've heard about using `Butler.from_config` to avoid the need for `type: ignore` here, right?  I don't really mind if you opt not to use it, but I wanted to make sure you were aware.

Reply:
Yeah, I remember, and when I heard about it there was a reason I picked this, but I can't now remember what that reason was. I also wonder if the `from_config()` Just Works with the `+sasquatch` stuff. I don't know if that was the reservation, but that's in the mix too.

Reply:
OK, seems to work so far, though I've not actually tried the `+sasquatch` part yet, but I've made the switch everywhere now 👍